src/main/java/com/goterl/lazycode/lazysodium/LazySodium.java

```diff
@@ -16,6 +16,7 @@
 import com.sun.jna.ptr.IntByReference;
 import com.sun.jna.ptr.PointerByReference;
 
+import javax.crypto.AEADBadTagException;
```

Comment:
As Lazysodium is available across Java and Android, I'd really rather not include a new package if possible.